wiki/History_of_osu!/2010/en.md

```diff
@@ -1 +1,68 @@
 # History of osu! 2010
+
+## January
+
+On the [Wai-con](https://en.wikipedia.org/wiki/Wai-Con) anime convention in Perth, osu! had an arcade booth which featured a fully self-contained arcade machine running a touch-based version of osu! on it called osu!arcade. Although playing too much on it could lead to sore fingers as some people had criticised, the prototype containing custom-made hardware and a built-in touchscreen was well-received by the community.[^wai-con]
+
+![A touchscreen displaying osu!'s main menu](img/osu!arcade.jpg "The osu!arcade machine at 2010's Wai-con")
+
+## February
+
+A new osu! update made small improvements in the overall user interface and added the ability to move all storyboard events by an arbitrary time value, allowing for more fine-tuned storyboard animations. In addition to this, usernames became clickable in chat logs.[^stable-b1460]
```

Comment:
while these are helpful UX improvements, they don't sound significant at all (at least to me)

Reply:
what I meant by this is that if an update has brought "small improvements" and did something mild to the storyboarding engine, it doesn't read as something worth bringing up at all, hence the comment